public/documentation/getting_started/remote_debugging.md

```diff
@@ -152,11 +152,11 @@ Note that it is also possible to disable remote debugging using the same intent,
 
 ## Tizen
 
-To debug Crosswalk applications on Tizen, the Crosswalk service on the target must be configured to provide a debugging endpoint. You can then access this endpoint from a Chrome browser on the host.
+To debug Crosswalk applications on Tizen, the Crosswalk service on the target device should be configured to provide a debugging endpoint.  A properly-configured endpoint is accessible via Chrome browser on the host. 
 
```

Comment:
Here you've changed "must" to "should".   In fact, it /must/ be /must/  :)
You also preferred to remove the article in the last sentence.  I'd prefer not having a lot of PRs that go about removing articles  ('the', 'an', 'a').  There's too much to do.   If the sentence makes sense already, leave it.